web/src/test/java/org/cbioportal/genome_nexus/web/AnnotationIntegrationTest.java

```diff
@@ -270,13 +270,26 @@ public void testGenomicLocationNucleotideContextPOST() {
         assertEquals("CAC", seq);
     }
 
+    @Test
+    public void testVariantAnnotationOriginalQuery() {
+        String expectedConvertedVariant = "4:g.55152096_55152107delTCATGCATGATT";
+        String genomicLocationString = "4,55152095,55152107,ATCATGCATGATT,A";
+        GenomicLocation[] genomicLocations = {
+            genomicLocationStringToGenomicLocation(genomicLocationString)
+        };
+        List<Map<String, Object>> response = this.fetchVariantAnnotationByGenomicLocationPOST(genomicLocations);
+        assertEquals(genomicLocationString, response.get(0).get("originalVariantQuery"));
+        assertEquals(expectedConvertedVariant, response.get(0).get("variant"));
+    }
```

Comment:
is `fetchVariantAnnotationByGenomicLocationPOST` the only endpoint where we will have `originalVariantQuery`?

It seems like we might need this anywhere where we are doing a POST regardless of input being genomic location or hgvs?

That said I think this if fine for a quick fix